src/Microsoft.AspNetCore.Razor.Language/Legacy/TagHelperParseTreeRewriter.cs

```diff
@@ -817,6 +817,14 @@ private static bool IsEndTag(Block tagBlock)
             return relevantSymbol.Type == HtmlSymbolType.ForwardSlash;
         }
 
+        private static bool IsComment(Span span)
```

Comment:
Mark this internal with a `// Internal for testing` and unit test

Reply:
> Otherwise would have to do the same for the other methods too, which is out of scope from this change.

Don't need to make it consistent. This is a new method on a legacy class that we should definitely make sure works on its own. In an ideal world we would have done that for all of the private methods in this class but at the time we chose to rely on integration tests to verify all of this works. We've definitely had to pay for that decision in test maintenance 😢